kcidb/monitor/output.py

```diff
@@ -31,12 +31,13 @@
 LOGGER = logging.getLogger(__name__)
 
 
-class NotificationMessage:
+class NotificationMessage:  # pylint: disable=too-many-instance-attributes
```

Comment:
Could you put this above the class declaration? It will be easier to spot then, and will match the existing convention.